pages/Announcements/Overview.md

```diff
@@ -30,18 +30,36 @@ Most serializations use outside standards, but some require additional clarifica
 
 ### hexadecimal
 
+Used to represent bytes.
+
 - MUST use 0-9,a-f representation
 - MUST be lowercase
 - MUST be prefixed with a `0x`
-- MUST NOT have spaces
-- MUST NOT have any zero padding
+- MUST NOT have spaces or separators
+- MUST have two characters per byte
 
 | Bytes | Invalid | Valid |
 | --- | --- | --- |
-| 2 | `0x0123` | `0x123` |
-| 2 | `123h` | `0x123` |
-| 8 | `0x0000000000000abc` | `0xabc` |
-| 32 | `0x00003e34c4325f4461b9355027b314f3eb56d31af549f7da7bd9ef1ce951651e` | `0x3e34c4325f4461b9355027b314f3eb56d31af549f7da7bd9ef1ce951651e` |
+| 2 | `0x123` | `0x0123` |
+| 2 | `123h` | `0x0123` |
+| 2 | `0x0ABC` | `0x0abc` |
+| 8 | `0xabc` | `0x0000000000000abc` |
+| 32 | `0x3e34c4325f4461b9355027b314f3eb56d31af549f7da7bd9ef1ce951651e` | `0x00003e34c4325f4461b9355027b314f3eb56d31af549f7da7bd9ef1ce951651e` |
+
+### decimal
+
+Used to represent integers.
+Strings are used to avoid issues with different implementations of numbers.
+
+- MUST use 0-9 representation
+- MUST NOT have spaces or separators
+- MUST be a string
```

Comment:
It might also be worth mentioning that the standard BigInt format with a trailing `n` is incorrect.

Reply:
I think we should at least add an example showing it's wrong.

Reply:
<img width="539" alt="image" src="https://user-images.githubusercontent.com/1252199/130142204-8ce8cc72-92d8-4daa-bf10-040c8d45531e.png">